test/suites/devlxd.sh

```diff
@@ -1,11 +1,11 @@
 test_devlxd() {
   ensure_import_testimage
 
-  # shellcheck disable=SC2164
-  cd "${TEST_DIR}"
-  go build -tags netgo -a -installsuffix devlxd ../deps/devlxd-client.go
-  # shellcheck disable=SC2164
-  cd -
+  (
```

Comment:
@monstermunchkin what was the reason for this change? Maybe add to commit msg?

Reply:
The reason for this change is shellcheck bothering me to make this change.